op-proposer/proposer/driver.go

```diff
@@ -296,6 +295,13 @@ func (l *L2OutputSubmitter) FetchDGFOutput(ctx context.Context) (*eth.OutputResp
 		return nil, false, fmt.Errorf("could not fetch output at current block number %d: %w", currentBlockNumber, err)
 	}
 
+	if claim == common.Hash(output.OutputRoot) {
+		l.Log.Debug("Skipping proposal: output root unchanged since last porposed game", "last proposed claim", claim, "output_root", output.StateRoot)
```

Comment:
```suggestion
		l.Log.Debug("Skipping proposal: output root unchanged since last proposed game", "last_proposed_root", claim, "output_root", output.OutputRoot)
```